src/scene/model.js

```diff
@@ -188,24 +188,7 @@ class Model {
      * }
      */
     generateWireframe() {
-        var i;
-        var mesh;
-
-        // Build an array of unique meshes in this model
-        var meshes = [];
-        for (i = 0; i < this.meshInstances.length; i++) {
-            mesh = this.meshInstances[i].mesh;
-            if (meshes.indexOf(mesh) === -1) {
-                meshes.push(mesh);
-            }
-        }
-
-        for (i = 0; i < meshes.length; ++i) {
-            mesh = meshes[i];
-            if (!mesh.primitive[RENDERSTYLE_WIREFRAME]) {
-                mesh.generateWireframe();
-            }
-        }
+        MeshInstance._prepareRenderStyleForArray(this.meshInstances, RENDERSTYLE_WIREFRAME);
```

Comment:
Does this function need to be public any more now that we have `renderStyle` property?

Reply:
I only added new renderStyle property to render component, but have not changes API on model component .. so this should stay?

Reply:
LOL, sorry, I thought this was on the RenderComponent for some reason. Please ignore.